dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -74,7 +74,7 @@ protected String parseCommandText(String commandText) {
                     paramName = "";
                 String dataType = rs.getString(2);
                 DbParameterAccessor dbp = createDbParameterAccessor(
-                        paramName,
+                        '"' + paramName + '"',
```

Comment:
For me when i look into code everything could be change if we modify NameNormaliser and little refactor code around it. As i look now my change doesn't affect weird function parameters. I want just implement this case, because we need to transform old weird naming schema to new one understandable views  and we want to have covered with tests ;-).